api/cli.md

```diff
@@ -0,0 +1,110 @@
+# Nucleus Command Line Interface
+
+This is the spec for the minimum CLI options that nucleus implementations should expose.
+
+Implementations may also expose platform, vm, and other sspecialty options not found in the other implementations in addition to these.
+
+## nucleus [options] <file or dir> [--] ...
+
+Execute nucleus with the provided options on the specified app bundle and provide it with arguments if specified.
+
+The app bundle may be either a directory, a Zip file, or a JS file.
+
+The following rules apply depending on what is provided:
+
+##### Directory
+
+The directory will be considered the app bundle and nucleus will attempt to execute the default `main.js` file.
+
+##### Zip file
+
+The zip file will be considered the app bundle and nucleus will read instead of it and attempt to execute the default main.js file.
```

Comment:
this sentence reads a bit off, not quite sure what the expected behavior would be